src/components/side_panel/chart/building_blocks/data_series/data_series.xml

```diff
@@ -1,6 +1,25 @@
 <templates>
   <t t-name="o-spreadsheet.ChartDataSeries">
-    <Section class="'o-data-series'" title="title">
+    <Section class="'o-data-series'">
+      <div class="d-flex flex-row o-section-title">
```

Comment:
Can be moved in the `title` slot I think, see `pivot_title_section.xml`  for an exemple.